security/pkg/nodeagent/sds/sdsservice_test.go

```diff
@@ -302,6 +311,9 @@ func TestStreamSecretsPush(t *testing.T) {
 	if len(sdsClients) != 0 {
 		t.Errorf("sdsClients, got %d, expected 0", len(sdsClients))
 	}
+	if _, ok := st.secrets.Load(key); ok != false {
+		t.Errorf("found cached secret after stream close, expected non-exist")
```

Comment:
nit: expected non-exist -> expected the secret to not exist

Reply:
updated/